tests/test_elbv2/test_elbv2.py

```diff
@@ -415,10 +415,8 @@ def test_create_target_group_and_listeners():
     response.get("LoadBalancers").should.have.length_of(0)
 
     # And it deleted the remaining listener
-    response = conn.describe_listeners(
-        ListenerArns=[http_listener_arn, https_listener_arn]
-    )
-    response.get("Listeners").should.have.length_of(0)
+    with pytest.raises(ClientError):
```

Comment:
Hi @bclodius, thanks for the PR!
Would you mind adding some assertions to the error that's returned?
```
with pytest.raises(..) as ex:
    pass
ex.value.response["Message"].should.equal(..)  # doing this out the top of my head, exact notation might be different
```

Reply:
Hey @bblommers great suggestion.  I have added an assertion for the error thrown and updated my PR.